nautobot/core/views/generic.py

```diff
@@ -948,6 +957,10 @@ def post(self, request, **kwargs):
         logger = logging.getLogger("nautobot.views.BulkEditView")
         model = self.queryset.model
 
+        relationships_errors = get_relationships_errors(request, model)
+        if len(relationships_errors) > 0:
+            return redirect(get_route_for_model(model, "list"))
```

Comment:
Same comment here about staying on the bulk-edit view instead of sending them back to the list view, reporting errors on the form if applicable.